src/main.rs

```diff
@@ -227,6 +229,30 @@ async fn main() -> ExitCode {
             Err(error) => {
                 eprintln!("Failed to load configuration: {error:?}");
                 eprintln!("Example configuration:\n\n{}", Config::default());
+
+                // Build temporary config to pass in wireserver defaults for report_failure
+                let cfg = Config::default();
+
+                let report_result = report_failure(
+                    &ReportableError::unhandled_exception().to_string(),
```

Comment:
this should be replaced with an error specific to this, e.g. reason=failure to load configuration and see what we can include from this error